<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DVisorEventos</class>
 <widget class="QDialog" name="DVisorEventos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visor Eventos</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>161</width>
     <height>113</height>
    </rect>
   </property>
   <property name="title">
    <string>Acci√≥n Doble click</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rBEliminar">
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBMostrar">
        <property name="text">
         <string>Mostrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBOcultar">
        <property name="text">
         <string>Ocultar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo Evento : </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoNuevoEvento">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Info NUEVOEVENTO</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>229</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Filtrar Por Tipo de Evento </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboFiltroTipo"/>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listaEventos">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>31</y>
     <width>281</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="vistaTablaEventos">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>581</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DVisorEventos</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DVisorEventos</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
